cni/pkg/nodeagent/cni-watcher.go

```diff
@@ -200,3 +186,32 @@ func (s *CniPluginServer) ReconcileCNIAddEvent(ctx context.Context, addCmd CNIPl
 
 	return nil
 }
+
+func (s *CniPluginServer) getPodWithRetry(log *istiolog.Scope, name, namespace string) (*corev1.Pod, error) {
+	log.Debugf("Checking pod: %s in ns: %s is enabled for ambient", name, namespace)
```

Comment:
```suggestion
	log.Debugf("Checking if pod %s/%s is enabled for ambient", name, namespace)
```

nit suggestion to better match the non-colon strings we use elsewhere in this PR.

Current:
> Checking pod: foo in ns: bar is enabled for ambient

Suggestion:
> Checking if pod foo/bar is enabled for ambient

Reply:
double nit, can we do `namespace/name` if we're going to make a change?